mrtt-ui/src/components/ProjectDetailsForm/ProjectDetailsForm.js

```diff
@@ -38,68 +37,54 @@ function ProjectDetailsForm() {
         })
       )
       .min(1)
-      .typeError('Select at least one item')
+      .typeError('Select at least one country')
   })
   const formOptions = { resolver: yupResolver(validationSchema) }
 
   // get functions to build form with useForm() hook
-  const { register, control, handleSubmit, formState, watch } = useForm(formOptions)
+  const { control, handleSubmit, formState, watch } = useForm(formOptions)
   const { errors } = formState
   const watchHasProjectEndDate = watch('hasProjectEndDate', 'false')
+  const [isSubmitting, setisSubmitting] = useState(false)
+  const [isError, setIsError] = useState(false)
+
+  const onSubmit = async (data) => {
+    setisSubmitting(true)
+    setIsError(false)
+    let preppedData = []
+    const url = 'https://mrtt-api-test.herokuapp.com/api/v2/sites/1/registration_answers'
```

Comment:
`https://mrtt-api-test.herokuapp.com/api/v2/` should be an environment variable. That way we can ensure it's the same throughout the app and can be updated based on the environment we deploy to. See [docs](https://create-react-app.dev/docs/adding-custom-environment-variables/) for more info.

This could be done as part of a refactor when writing the next form.

Reply:
agreed! will start an env file